components/03-modules/header/_search-form.scss

```diff
@@ -1,29 +1,33 @@
-$search-form__height               : 34px !default;
-$search-form__input-height         : 32px !default;
-$search-form__border-color         : $gray-dark !default;
-$search-form__border-color--xl     : $gray !default;
-$search-form__font-size            : $font-size-small !default;
-$search-form__placeholder-color    : $gray-dark !default;
-$search-form__placeholder-color--xl: $gray !default;
-$search-form__button-size          : 18px !default;
-$search-form__button-background    : transparent !default;
-$search-form__button-icon-color    : $gray-dark !default;
+$search-form__height           : 34px !default;
+$search-form__width            : 100% !default;
+$search-form__width--xl        : 70% !default;
+$search-form__alignment        : flex-end !default;
+$search-form__margin           : 0 !default;
+$search-form__border-color     : $gray-light !default;
+$search-form__font-size        : $font-size-small !default;
+
+$search-form__input-height     : 32px !default;
+$search-form__input-color      : $gray !default;
+
+$search-form__button-size      : 18px !default;
+$search-form__button-margin    : 0 !default;
+$search-form__button-border    : none !default;
+$search-form__button-background: transparent !default;
+$search-form__button-icon-color: $gray-dark !default;
+
+$search-form__placeholder-color: $gray !default;
 
 .search-form {
     position: relative;
-    width: 100%;
+    width: $search-form__width;
     height: $search-form__height;
-    margin: 0;
-    align-self: center;
-    border: 1px solid $search-form__placeholder-color;
+    align-self: $search-form__alignment;
+    margin: $search-form__margin;
+    border: 1px solid $search-form__border-color;
     border-radius: $search-form__height;
     z-index: 1;
-    @include mq($screen-m) {
-        width: 70%;
-    }
     @include mq($screen-xl) {
-        width: 100%;
-        border: 1px solid $search-form__border-color--xl;
+        width: $search-form__width--xl;
     }
 
     &__label {
```

Comment:
instead of `display: none` please use `label--hidden` styles please

Reply:
Added class label--hidden